xonsh/completers/path.py

```diff
@@ -185,7 +185,7 @@ def _quote_paths(paths, start, end, append_end=True, cdpath=False):
         expanded = expand_path(s)
         if os.path.isdir(expanded) or (cdpath and _is_directory_in_cdpath(expanded)):
             _tail = slash
-        elif end == "":
+        elif end == "" and not s.endswith("="):
             _tail = space
```

Comment:
does the completion get generated from path-completer or the bash-completer ?

Reply:
@jnoortheen it gets generated from the bash-completer:
https://github.com/xonsh/xonsh/blob/bff0794b68f0d9266d8d920f10170c982927ccf6/xonsh/completers/bash_completion.py#L425-L426

Reply:
the module bash-completer already has a quote-paths function https://github.com/xonsh/xonsh/blob/09a12e4e85f331d0d281aafacd2d7f69558d7014/xonsh/completers/bash_completion.py#L167  . Is that redundant ? 
also a small question, will this affect other completions ?

Reply:
> will this affect other completions ?

Good catch! It seems like file paths ending with an equal sign will now be completed without a trailing space (I've added a failing test to verify this)

Perhaps a better approach would be to restore `_quote_paths` to the original and only apply the equal sign fix to `_bash_quote_paths`.

This way, regular paths ending with an equal sign would be correctly completed _with_ a trailing space, and bash command arguments that use an equal sign would be correctly completed _without_ a trailing space.

Reply:
also, should `tests/test_path_completers.py` be moved to `tests/completers/test_path_completers.py`? (this isn't really related to this PR though)

Reply:
yes moving that file will be good.

Reply:
> Perhaps a better approach would be to restore _quote_paths to the original and only apply the equal sign fix to _bash_quote_paths.

yes that will be good.